docs/source/api/references/graphstorm.trainer.rst

```diff
@@ -3,11 +3,11 @@
 graphstorm.trainer
 =====================
 
-    GraphStorm trainers assemble the distributed training pipeline for different tasks or
-    different training methods.
+    GraphStorm Trainers assemble the distributed training pipeline for different tasks,
+    e.g., node classification and link prediction.
 
-    If possible, users should always use these trainers to avoid handling the distributed
-    processing and tasks.
+    If possible, users should always use these Trainers to avoid handling the complexities
+    involved with the distributed data processing and model training.
```

Comment:
Why trainers are related to distributed data processing?
I think originally, the distributed processing means distributed data loading, etc.